epochStart/bootstrap/process.go

```diff
@@ -247,6 +247,17 @@ func (e *epochStartBootstrap) Bootstrap() (Parameters, error) {
 		return e.prepareEpochZero()
 	}
 
+	e.dataPool, err = factoryDataPool.NewDataPoolFromConfig(
```

Comment:
maybe move only where it is used. It can stay like this, it is not cricital

Reply:
keeping this here will ensure that the next calls to interceptor container will have the data pool, it will be less prone to errors :)